### ISSUE TITLE
version 1.2-beta2 package 2 times the root project as jar

### ISSUE THREAD

**Issue by yanns** [play2war/play2-war-plugin#205]
With a multi-modules project like that one:

``` scala
lazy val core = playProject("core")

lazy val root = playProject("myapp").in(file("."))
    .dependsOn(core)
    .aggregate(core)

private def playProject(name: String) =
    play.Project(
      name = name,
      applicationVersion = "1",
      path = file("modules/" + name)
    )
```

the root project is packaged 2 times as jar in the generated war:
- one time as myapp.myapp-1.jar
- one time as myapp_2.10-1.jar

(You can assign this issue to me)

**Comment by dlecan** [comment]
Fixed by #212

**Comment by dlecan** [comment]
Play2War v1.2-beta4 with a fix for this issue is out!
